.github/workflows/build.yml

```diff
@@ -8,17 +8,40 @@ on:
 
 jobs:
   buildx:
+    name: Build Docker Image
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Check pushing to Docker Hub
+        id: push-other-places
+        # Only push to Dockerhub from the main repo
+        # Otherwise forks would require a Docker Hub account and secrets setup
+        run: |
+          if [[ ${{ github.repository_owner }} == "caronc" ]] ; then
+            echo "Enabling DockerHub image push"
+            echo "enable=true" >> $GITHUB_OUTPUT
+          else
+            echo "Not pushing to DockerHub"
+            echo "enable=false" >> $GITHUB_OUTPUT
+          fi
+
+      # Mostly for forks, set an output package name for ghcr.io using the repo name
+      - name: Set ghcr repository name
```

Comment:
Not strictly speaking necessary, but makes it easier for forks to build the image and push it to the ghcr.io, so you can just pull an image to test.

Gives an image input [like this](https://github.com/stumpylog/apprise-api/actions/runs/12510858521/job/34902151644#step:5:25) on a fork and should have both tags enabled on the main repo.